homework/content.js

```diff
@@ -1,37 +1,39 @@
-'use strict';
-
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
-
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
-
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
-
-  function drawNote(note) {
-    var title = note.title;
-    var h = document.createElement('h2');
-    h.textContent = title;
-    var passages = note.passages;
-    var buff = document.createDocumentFragment();
-    passages.forEach(function(passage) {
-      var p = document.createElement('p');
-      p.classList.add('note-passage');
-      p.textContent = passage;
-      buff.appendChild(p);
-    });
-    _wrapper.appendChild(h);
-    _wrapper.appendChild(buff);
-  }
-
-  document.addEventListener('DOMContentLoaded', function(event) {
-    start();
-  });
-})();
+'use strict';
+
+function Content() {
+	this._wrapper = document.querySelector('#note-content-wrapper');
+}
+
+Content.prototype = {
+
+	start : function () {
+		window.addEventListener('note-open', (function(event) {

```

Comment:
Instead of using an anonymous function, set the listener as `this` and handle events with `handleEvent` method is better.